### PR TITLE
Create a chennai file

### PR DESCRIPTION
**Pull request by passion4aix**
This PR closes issue #20 after adding a file about Chennai, India

Give me a thums up ;)

### CODE REVIEW

**Comment by passion4aix** [comment]
:+1:

File: passion4aix-chennai-data.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Chennai-India
+
+### Place to eat
+
```
Comment (passion4aix) [inline passion4aix-chennai-data.md]: what are these... i'd love to visit them :100:

Comment (passion4aix) [inline passion4aix-chennai-data.md]: :1234:

Comment (Peripona) [inline passion4aix-chennai-data.md]: Do What you Like and Love what you DO.. 👋

Comment (passion4aix) [inline passion4aix-chennai-data.md]: absolutely :100:

**Pull request merged**